stacks-signer/src/runloop.rs

```diff
@@ -234,12 +234,23 @@ impl From<&Config> for RunLoop<FrostCoordinator<v2::Aggregator>> {
             .iter()
             .map(|i| i - 1) // Signer::new (unlike Signer::from) doesn't do this
             .collect::<Vec<u32>>();
+        // signer uses a Vec<u32> for its key_ids, but coordinator uses a HashSet for each signer since it needs to do lots of lookups
+        let mut signer_key_ids = HashMap::new();
+        for (signer_id, key_ids) in &config.signer_key_ids {
+            let ids = key_ids.iter().map(|i| *i - 1).collect::<HashSet<u32>>();
```

Comment:
Won't this be off by one? We already do -1 above in key_ids. This essentially will be doing signer_key_ids - 2 now, no? Might be misunderstanding the logic.

Reply:
No, because `key_ids` here is an ephemeral iterator for this loop.  It's not the `key_ids` variable declared above.  Yay scoping!

I'll change the loop iterator name so it's not confusing.

Reply:
I ended up doing a nested map, it's cleaner and that's the style the `fn` uses anyway.